helpers.js

```diff
@@ -1,19 +1,33 @@
 const util = require("util");
 const childProcess = require("child_process");
-const { access } = require("fs/promises");
-const fs = require("fs");
+const fs = require("fs/promises");
 
 const exec = util.promisify(childProcess.exec);
 
 async function assertPathExistence(path) {
```

Comment:
This function no longer asserts that the path exists, but it checks if a path exists and returns a boolean, you should rename this function to `doesPathExist` or `pathExists`.

Reply:
now it returns the type path - directory, file, or nonexistent.